lib/main.coffee

```diff
@@ -2,6 +2,16 @@
 helpers = require 'atom-linter'
 path = require 'path'
 
+isGreaterVersion = (v1, v2) ->
+  v1Parts = v1.split('.')
+  v2Parts = v2.split('.')
+  minLength = Math.min(v1Parts.length, v2Parts.length)
+  if minLength > 0
+    for idx in [0..minLength - 1]
+      unless Number(v1Parts[idx]) == Number(v2Parts[idx])
+        return Number(v1Parts[idx]) < Number(v2Parts[idx])
+  return v1Parts.length < v2Parts.length
```

Comment:
I did some tests and it seems to be this should be `=`

Tests: https://pastebin.com/2UY4pA1E
You can make test online here: http://js2.coffee/